### PR TITLE
ie11 memory leaks

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @MargaritaLoseva

### CODE REVIEW

File: src/client/sandbox/xhr/index.js [hunk 1]
```diff
@@ -117,9 +117,13 @@ export default class XhrSandbox extends SandboxBase {
...
-            XMLHttpRequestWrapper.prototype = xhr;
+            XhrSandbox.XMLHttpRequestWrapper.prototype = xhr;
 
-            return new XMLHttpRequestWrapper(xhr);
+            return new XhrSandbox.XMLHttpRequestWrapper(xhr);
         };
     }
 }
+
+// HACK: Do not even try to understand why it's done. Believe me,
```
Comment (MargaritaLoseva) [inline src/client/sandbox/xhr/index.js]: Changed: Do not even try to understand why _this has been done_.

**Comment by churkin** [comment]
@inikulin I have found a simple fix. We do not need to rewrite wrapper prototype as we add all the methods and properties manually. And it solves the problem.

**Comment by inikulin** [comment]
That's cool. But we did it actually?

**Comment by churkin** [comment]
I did not understand. It looks like a very old code.

**Comment by churkin** [comment]
https://gist.github.com/churkin/e3a4fdd29010cb532572

**Comment by churkin** [comment]
FPR

**Comment by inikulin** [comment]
I guess it was added to keep properties that are not covered because of this filter: https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/xhr/xml-http-request-wrapper.js#L68 

I'd like to investigate this one a little better. At least try to find this proto assignment was introduced.

**Comment by MargaritaLoseva** [comment]
lgtm

**Comment by churkin** [comment]
@inikulin in this case 
```javascript
if (!Object.prototype.hasOwnProperty(prop)) {
```
check is unnecessary. It does not apply to prototype, always will return true to the XMLHttpRequest.

**Comment by inikulin** [comment]
Ok, let's try it as is and rollback in case of issues in HHM

**Pull request merged**